static/src/server.ts

```diff
@@ -174,47 +173,47 @@ function getTileResult(
   switch (tileConfig.type) {
     case "LINE":
       return getLineTileResult(
-          id,
-          tileConfig,
-          place,
-          svSpec,
-          CONFIG.apiRoot,
-          "",
-          false
-        )
+        id,
+        tileConfig,
+        place,
+        svSpec,
+        CONFIG.apiRoot,
+        "",
+        false
+      );
```

Comment:
These changes are all from running lint.

Reply:
@juliawu can you help fix the lint issue in cloudbuild?